.gitignore

```diff
@@ -7,3 +7,5 @@ django
 celerybeat-schedule/
 
 .venv
+
+.idea
```

Comment:
Usually I leave editor configs out and let people deal with that locally (e.g. a global gitignore), but since we as a community usually recommend pycharm this is good.

You should probably also add `*.iml` for older versions of PyCharm (before they moved it into the `./idea` dir).